qiskit/circuit/library/standard_gates/u3.py

```diff
@@ -52,6 +65,11 @@ class U3Gate(Gate):
 
     **Examples:**
 
+    .. math::
+
+        U3(\theta, \phi, \lambda) = e^{i \frac{\pi + \theta}{2}} P(\phi + \pi) \sqrt{X}
+        P(\theta + \pi) \sqrt{X} P(\lambda)
```

Comment:
Also here the global phase doesn't seem to match (or maybe I have a typo!)

```python
>>> decomp = np.exp(0.5j * (np.pi + th)) * np.dot(PhaseGate(phi + np.pi), np.dot(SXGate(), np.dot(PhaseGate(th + np.pi), np.dot(SXGate(), PhaseGate(lam)))))
>>> decomp
array([[-0.53350621-0.71685923j,  0.22160969+0.39034999j],
       [ 0.23851082-0.38025854j,  0.56442266-0.69277925j]])
>>> u3 = U3Gate(th, phi, lam).to_matrix()
>>> u3  # matrix doesn't match decomp
array([[ 0.89359724+0.j        , -0.44545363-0.05527232j],
       [ 0.1626514 +0.41836406j,  0.21878112+0.86640109j]])
>>> Operator(u3).equiv(decomp)  # but it is equivalent -- hence a global phase difference exists
True
```

Reply:
That's because I forgot to add a minus sign to the global phase 😅. The correct phase is $e^{-i \frac{\theta + \pi}{2}}$.